packages/grader/src/task/GradeTask.ts

```diff
@@ -75,11 +75,9 @@ export class GradeTask {
 
                 if (this.containerState === "TIMEOUT") {
                     out.report.feedback = "Container did not complete in the allotted time.";
-                    out.postbackOnComplete = true;
                     out.state = "TIMEOUT";
                 } else {
                     try {
-                        // TODO Verify that the report is actually valid
                         out.report = await this.workspace.readJson("output/report.json");
                         out.postbackOnComplete = exitCode !== 0;
```

Comment:
@nickbradley  this is just being pedantic, but i wonder if we want this to be just .25% more readable (aka by assigning exitCode!==0 to an intermediate var)